packages/blade/src/components/Dropdown/dropdownUtils.ts

```diff
@@ -0,0 +1,244 @@
+/*
+ * This content is licensed according to the W3C Software License at
+ * https://www.w3.org/Consortium/Legal/2015/copyright-software-and-document
+ *
+ * This software or document includes material copied from or derived from
+ * https://www.w3.org/WAI/ARIA/apg/example-index/combobox/combobox-select-only.html.
```

Comment:
Most of the code in this file is taken from w3.org's example

Reply:
confession 🤣